src/uwtools/tests/config/test_validator.py

```diff
@@ -128,18 +129,121 @@ def test_validate_yaml_pass(config, config_file, schema, schema_file):
     assert validator.validate_yaml(schema_file=schema_file, config_file=config_file)
 
 
-@pytest.mark.parametrize(
-    "vals", [("hello_workflow_tags.yaml", True), ("hello_workflow_invalid.yaml", False)]
-)
-def test_validate_workflow_tags_pass(vals):
-    fn, validity = vals
+@fixture
+def rocoto_assets():
     with resources.as_file(resources.files("uwtools.resources")) as resc:
         schema_file = resc / "rocoto.jsonschema"
-    config_file = support.fixture_path(fn)
-    assert (
-        validator.validate_yaml(schema_file=schema_file, config_file=config_file, check_path=False)
-        is validity
-    )
+    kwargs = {
+        "schema_file": schema_file, 
+        "config_file": "/not/used", 
+        "check_paths": False
+    }
+    config = {
+        "workflow": {
+            "cycledefs": {"howdy": ["202209290000 202209300000 06:00:00"]},
+            "log": "/some/path/to/&FOO;",
+            "tasks": {
+                "metatask": {
+                    "var": {"member": "foo bar baz"},
+                    "task": {
+                        "cores": 88,
+                        "command": "some-command",
+                        "walltime": "00:01:00"
+                    }
+                }
+            }
+        }
+    }
+    return kwargs, config
+
+
+def test_validate_yaml_rocoto_invalid_dependency(rocoto_assets, caplog):
+    kwargs, config = rocoto_assets
+    config["workflow"]["tasks"]["metatask"]["task"].update({"stdout": "hello"})
```

Comment:
Here I added a new test case on the `dependencies` for `stderr` and `stdout`. When replacing with `dependentRequired` I see we pass when we should not be. Having trouble tracking this one down and fully understanding why it's not a 1:1 replacement for each other.